src/Common/Types.ts

```diff
@@ -296,33 +326,57 @@ export interface MaterialInfo {
     type?: string
 }
 
-export interface CDMaterialType {
-    index: number
-    id: string
-    materialInfo: MaterialInfo[]
-    tab: CDModalTabType
-    scanEnabled: boolean
-    scanned: boolean
-    vulnerabilitiesLoading: boolean
-    lastExecution: string //timestamp
-    vulnerabilities: VulnerabilityType[]
-    vulnerable: boolean
-    deployedTime: string
-    deployedBy?: string
-    wfrId?: number
-    buildTime: string
-    image: string
-    isSelected: boolean
-    showSourceInfo: boolean
-    latest: boolean
-    runningOnParentCd?: boolean
-    userApprovalMetadata?: UserApprovalMetadataType
-    triggeredBy?: number
-}
-
 export interface CDMaterialResponseType {
     approvalUsers: string[]
     materials: any[]
     userApprovalConfig: UserApprovalConfigType
     requestedUserId: number
-}
\ No newline at end of file
+    tagsEditable: boolean
+    appReleaseTagNames: string[]
+}
+
+export interface ReleaseTag {
```

Comment:
we should move release tag related types to separate type file

Reply:
done